api-report/cell.api.md

```diff
@@ -16,11 +16,31 @@ import { ISummaryTreeWithStats } from '@fluidframework/runtime-definitions';
 import { Serializable } from '@fluidframework/datastore-definitions';
 import { SharedObject } from '@fluidframework/shared-object-base';
 
+// @public
+export interface AttributionKey {
+    seq: number;
+    type: "op";
+}
+
+// @public
+export interface ICellAttributionOptions {
+    // (undocumented)
+    track?: boolean;
+}
+
+// @public
+export interface ICellOptions {
+    // (undocumented)
+    attribution?: ICellAttributionOptions;
+}
+
 // @public
 export interface ISharedCell<T = any> extends ISharedObject<ISharedCellEvents<T>> {
     delete(): void;
     empty(): boolean;
     get(): Serializable<T> | undefined;
+    // (undocumented)
+    getAttribution(): AttributionKey | undefined;
```

Comment:
i'd mark this @alpha and document it here. Then the implementation can just use @inheritdoc (and can mark alpha as well to be clear)